CompatibilityRequirements.md

```diff
@@ -184,8 +187,8 @@ were a listener or callback interface) then this is never permissible
 will be no or few wrapper implementations of the interface, or if they
 would always be provided as an add-on by the developers of vendor
 implementations, it may be permissible to add a new method to an
-existing interface after careful consideration by the specification's
-expert group  (see Rule 3 for a full description).
+existing interface after careful consideration by the specification
+project team  (see Rule 3 for a full description).
```

Comment:
```suggestion
project  (see Rule 3 for a full description).
```

As above: In most cases, you changed "specification's expert group" to "specification project".